.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Thank you for contributing to JanusGraph.
+
+In order to streamline the review of the contribution we ask you
+to ensure the following steps have been taken:
+
+### For all changes:
+- [ ] Is there an issue associated with this PR? Is it referenced 
+     in the commit message?
+
+- [ ] Does your PR title ends with #xyz where xyz is the issue number you are trying to resolve?
```

Comment:
Ok, I misunderstood you here, after my last change it now says "PR body" instead of "PR title".